pkg/cloud/ovirt/machine/actuator.go

```diff
@@ -496,6 +496,25 @@ func validateVirtualMachineType(vmtype string) *apierrors.MachineError {
 	return nil
 }
 
+// validateHugepages execute validation regarding the Virtual Machine hugepages
+// custom property (2048, 1048576).
+// Returns: nil or InvalidMachineConfiguration
+func validateHugepages(value int32) *apierrors.MachineError {
+	if value == 0 {
+		return nil
+	}
+	switch value {
+	case 2048, 1048576:
+		return nil
+	default:
+		return apierrors.InvalidMachineConfiguration(
```

Comment:
return an error and wrap it with apierrors.InvalidMachineConfiguration on the caller :)